Split/Network/Targets/Endpoint.swift

```diff
@@ -0,0 +1,61 @@
+//
```

Comment:
Shouldn't we rename folder to `/Network/Endpoints/...`? instead of `/Network/Targets/...`

Reply:
Done Doc!